internal/datacoord/handler.go

```diff
@@ -48,7 +48,7 @@ func (h *ServerHandler) GetVChanPositions(channel string, collectionID UniqueID,
 		}
 
 		if s.GetState() == commonpb.SegmentState_Dropped {
-			dropped = append(dropped, s.GetID())
+			dropped = append(dropped, s.SegmentInfo)
```

Comment:
Dropped `SegmentInfo` need to be trimmed, preventing vchannel info being to large

Reply:
fixed